.drone.yml

```diff
@@ -7,6 +18,7 @@ services:
     image: fpfis/httpd-php-dev:${PHP_VERSION=5.6}
     environment:
     - DOCUMENT_ROOT=/test/atomium
+    pull: true
```

Comment:
I think you forget this one, isn't it?

Reply:
Nope, @voidtek told me that the `pull: true` must be in the `services`, not in the `pipeline`.